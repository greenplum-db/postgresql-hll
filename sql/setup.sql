CREATE EXTENSION hll VERSION '2.16';

-- Using random distribution policy by default when creating tables
ALTER database contrib_regression
  SET gp_create_table_random_default_distribution TO 'on';

-- ALTER EXTENSION hll UPDATE TO '2.11';
-- ALTER EXTENSION hll UPDATE TO '2.12';
-- ALTER EXTENSION hll UPDATE TO '2.13';
-- ALTER EXTENSION hll UPDATE TO '2.14';
-- ALTER EXTENSION hll UPDATE TO '2.15';
-- ALTER EXTENSION hll UPDATE TO '2.16';
