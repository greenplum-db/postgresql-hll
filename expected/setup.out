CREATE EXTENSION hll VERSION '2.16';
-- We only need the latest sql file when release. So there is no need to test
-- the upgrade SQL file
-- ALTER EXTENSION hll UPDATE TO '2.11';
-- ALTER EXTENSION hll UPDATE TO '2.12';
-- ALTER EXTENSION hll UPDATE TO '2.13';
-- ALTER EXTENSION hll UPDATE TO '2.14';
-- ALTER EXTENSION hll UPDATE TO '2.15';
-- ALTER EXTENSION hll UPDATE TO '2.16';
